### ISSUE TITLE
Can't start server if new App Engine developer Console is used to create account

### ISSUE THREAD

**Issue by jackpal** [mc-coal/mc-coal#106]
I tried installing MC Coal on a new App Engine / GCE account today, using the current developer console.

When I tried to bring up my server I ran into the error, below [1].

I guess GCE has changed so that a Google Cloud Storage bucket has to be set up manually for new projects. Maybe you should add a documentation step telling people to configure a default GCS bucket name. And probably you should add a runtime check that that app_identity.get_default_gcs_bucket_name() returns a non-None bucket name. (Since it will be None by default.)

See this stack overflow post for discussion:

http://stackoverflow.com/questions/24236825/my-app-engine-app-is-not-automatically-creating-default-bucket

I was able to work around this issue by following the instructions in the stack overflow post (to visit the old app engine console settings page and manually enable "Cloud Integration".)

[1]
Traceback (most recent call last):
  File "/base/data/home/apps/s~minecraft-mc-coal/1.390502301210073583/server_queue.py", line 24, in start_server
    instance.start()
  File "/base/data/home/apps/s~minecraft-mc-coal/1.390502301210073583/gce.py", line 89, in start
    verify_bucket(num_versions=self.backup_depth)
  File "/base/data/home/apps/s~minecraft-mc-coal/1.390502301210073583/gcs.py", line 107, in verify_bucket
    request = service.buckets().get(bucket=bucket)
  File "/base/data/home/apps/s~minecraft-mc-coal/1.390502301210073583/lib/apiclient/discovery.py", line 583, in method
    raise TypeError('Missing required parameter "%s"' % name)
TypeError: Missing required parameter "bucket"

**Comment by gumptionthomas** [comment]
Good catch! Obviously google has changed the preferred method for setting up a Google Cloud (app engine) Project since I wrote the instructions.

I'll update the install docs to indicate that it is necessary to create the default bucket with the name `[my-project-id].appspot.com` by following the instructions here: https://cloud.google.com/appengine/docs/python/googlecloudstorageclient/activate
